narwhals/_spark_like/expr.py

```diff
@@ -233,6 +256,50 @@ def _min(_input: Column) -> Column:
 
         return self._from_call(_min, "min", returns_scalar=True)
 
+    def null_count(self) -> Self:
+        def _null_count(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.count_if(F.isnull(_input))
+
+        return self._from_call(_null_count, "null_count", returns_scalar=True)
+
+    def replace_strict(
```

Comment:
I am tempted to say that this should not be implemented for now and just raise a `NotImplementedError`. (as we do in Dask)
We would need to be able to access the dataframe (and collect the results) to get the distinct values of the column.

@FBruzzesi and @MarcoGorelli any thoughts?

Reply:
> I am tempted to say that this should not be implemented for now and just raise a NotImplementedError. (as we do in Dask)

Sure we can evaluate if and how to support `replace_strict` later on. Super excited to ship the rest for now 🙌🏼

Reply:
agree!